cmd/mo-service/main.go

```diff
@@ -50,7 +50,7 @@ import (
 
 var (
 	configFile = flag.String("cfg", "", "toml configuration used to start mo-service")
-	launchFile = flag.String("launch", "./etc/launch-tae-multi-CN-tae-DN/launch.toml", "toml configuration used to launch mo cluster")
+	launchFile = flag.String("launch", "", "toml configuration used to launch mo cluster")
```

Comment:
should we raise an error if both `-cfg` and `-launch` are not set?

Reply:
added